extensions/amp-timeago/0.1/lib/timeago/timeago.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This is a fork of the timago.js library developed by hustcc
+ * https://github.com/hustcc/timeago.js
```

Comment:
For open source compliance reasons I must ask you to

- rename `lib` to `third_party`
- not change any license related code or add licenses to forked code
- include the original license file
- add a README.amp file briefly describing modifications.